src/script/conversation/ConversationRepository.ts

```diff
@@ -1363,9 +1363,7 @@ export class ConversationRepository {
         this.eventRepository.injectEvent(response, EventRepository.SOURCE.BACKEND_RESPONSE);
       }
     } catch (error) {
-      if (error instanceof BackendClientError) {
-        this.handleAddToConversationError(error, conversationEntity, userIds);
-      }
+      this.handleAddToConversationError(error as BackendClientError, conversationEntity, userIds);
```

Comment:
type narrowing with `instanceof` doesn't work in this case?

Reply:
no it was silencing errors because we want to catch all regardless of whether they conform to this type, i could alternately just call this error an any

Reply:
but i hate any

Reply:
ok then, but in the future we should take care about typing all the errors without casting